src/crunker.ts

```diff
@@ -58,9 +58,25 @@ export default class Crunker {
   }
 
   /**
-   * Asynchronously fetches multiple audio files and returns an array of AudioBuffers.
+   * Asynchronously fetches multiple audio files and returns an array of AudioBuffers,
+   * splitting into groups of 200 files to avoid resource exhaustion.
    */
   async fetchAudio(...filepaths: CrunkerInputTypes[]): Promise<AudioBuffer[]> {
+    const buffers: AudioBuffer[] = [];
```

Comment:
I never really did expose proper batching and retry mechanisms here. Perhaps we should expose the `200` as a user configurable value with this 100 or 200 as a "sensible" default?

This way it can be overridden if required.

Reply:
Sounds good. Probably in the Crunker constructor would make the most sense